src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+// import TodoContainer from './classBased/components/TodoContainer';
+import { BrowserRouter } from 'react-router-dom';
+import TodoContainer from './functionBased/components/TodoContainer';
+// import './classBased/App.css';
```

Comment:
- [ ] Kindly remove unnecessary imports you don't need from your code base to make it cleaner and more professional. This is a good practice and a nice habit to adopt.